static/js/util.js

```diff
@@ -493,6 +494,38 @@ function randDomId() {
     .substr(2, 10);
 }
 
+/**
+ * add or delete statvars from url
+ *
+ * @param {string} dcid of statvar
+ * @param {boolean} add = True, delete = False
+ * @return {string} The line chart URL.
+ */
+
+function updateUrlStatsVar(statvar, add){
```

Comment:
this would be a good case to add unit test

Reply:
done.